src/lib/features/onboarding/onboarding-read-model-type.ts

```diff
@@ -9,6 +9,17 @@ export type InstanceOnboarding = {
     firstLive: number | null;
 };
 
+/**
+ * All the values are in minutes
+ */
+export type ProjectOnboarding = {
+    project: string;
+    firstFeatureFlag: number | null;
+    firstPreLive: number | null;
+    firstLive: number | null;
+};
+
 export interface IOnboardingReadModel {
     getInstanceOnboardingMetrics(): Promise<InstanceOnboarding>;
+    getProjectsOnboardingMetrics(): Promise<Array<ProjectOnboarding>>;
```

Comment:
Why Array instead of [], what we usually have.

Reply:
I use both. I can fix in the next PR